### ISSUE TITLE
Auto updater

### ISSUE THREAD

**Issue by thomascullen** [thomascullen/voicebox#1]
Need to implement an auto updater which will update in the background. Releases should be pulled from GH releases.

[electron-gh-releases](https://github.com/jenslind/electron-gh-releases)

**Comment by brock** [comment]
Would also like to see some kind of indicator in the interface that shows that an update is available. Here is a rough mockup of the idea, but would be nice to also clue the user with something like "say 'update voicebox' to update":

<img width="338" alt="screen_shot_2015-10-07_at_7_31_52_pm" src="https://cloud.githubusercontent.com/assets/654725/10357297/20a25034-6d46-11e5-831a-38748ec404f2.png">

EDIT: Now that I think of it, possibly just replacing the text of "What can I do for you?" with "Say 'Update Voicebox' to get the latest update".

**Comment by thomascullen** [comment]
Yeah I think thats a good idea. I had implemented basics updates but it hits the Github open API limits pretty quick so would need to register a Github app to continue with it. Haven't got around to looking into what the best way to handle environment variables are in electron just yet.

I think maybe displaying a small piece of text at the very bottom under the sound wave graphic. And the text should maybe say 'A new update is available for VoiceBox. Say 'Update VoiceBox' to upgrade to the latest version'.
